pom.xml

```diff
@@ -34,6 +34,8 @@
   </licenses>
 
   <properties>
+    <!-- TODO: remove before merging -->
+    <enforcer.skip>true</enforcer.skip>
```

Comment:
Required just for testing the pipeline. Enforcer fails the build due to the http:// protocol of the testing Nexus server (https:// required).

Reply:
Yes, this must not make it to master.

Reply:
Maybe you can use `-Denforcer.skip=true` in the pipeline instead?

Reply:
this would be something similar to: https://github.com/kiegroup/kogito-cloud-operator/pull/618

Reply:
That's an option too, thanks.

Reply:
+1 for doing it in the pipeline which also adds the http:// url, instead of doing it in the pom.xml